homeassistant/components/lcn/__init__.py

```diff
@@ -172,6 +194,29 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     return True
 
 
+async def async_migrate_entities(
```

Comment:
Instead of calling this function on each setup, can we call it once during the config entry migration?